aulaglobal.py

```diff
@@ -61,7 +61,9 @@
 # Check if success
 url = br.open(BASE_URL)
 login = url.get('X-Frame-Options', None)
-status, _ = cgi.parse_header(login)
+m = Message()
+m['content-type'] = login
```

Comment:
You don't want to extract the `content-type` header here. This seems like a copy-paste from the [docs](https://peps.python.org/pep-0594/#cgi)

Reply:
The content type works on this section, if the password is wrong, it exits the script.
I can check other headers, but this one works.